lua/gk.lua

```diff
@@ -0,0 +1,21 @@
+local ffi = require("ffi")
+local gk_config = require("gk_conf")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- Function that sets up the GK functional block.
+function M.setup_block()
+	local ret = 0
+
+	-- Init the GK configuration structure.
+	conf = gatekeeperc.alloc_and_return_gk_conf()
+	conf.lcore_start_id = gk_config.lcores.lcore_start_id
```

Comment:
OK, I see what you're saying. I was thinking that such a feature would be useful for debugging and testing when you want to incrementally add new features via the static config, but the code doesn't need to have that complexity in all cases. The Dynamic Config will also help in incrementally testing new features.